boards/adi/max32657evkit/Kconfig.defconfig

```diff
@@ -47,6 +47,11 @@ choice TFM_PARTITION_LOG_LEVEL
 	default TFM_PARTITION_LOG_LEVEL_SILENCE
 endchoice
 
+# Download MCUBoot version that linked with TF-M
+choice TFM_MCUBOOT_PATH
+	default TFM_MCUBOOT_PATH_DOWNLOAD
+endchoice
```

Comment:
no

Reply:
Ok removed, but isn't it cause a sync issue? TF-M linked with different point of MCUBoot project. If tf-m uses mcuboot that comes with zephyr it might cause some build or execution issue.